index.html

```diff
@@ -110,6 +110,195 @@ <h1 class="name-text">Hey There. <br>
         </div>
       
         </section>
-        </div>
-    </body>
+
+        <section class="works">
+          <div id="works-title">
+           <h2 class="works-title-text">My Recent Works</h2>
+           <img src="images/line.svg" id="works-title-img"  alt="my works profile">
+          </div>
+          <div class="card-1">
+                <div id="div-1--left">
+                <img id="card-1-img" src="images/recenttitle.svg" alt="recent works">
+                </div>
+                <div id="div-1--right">
+                <h2 class="works-title-text" id="works-title-text-1">Multi-Post Stories</h2>
+                <p id="card-1-text">A daily selection of privately personalized reads;
+                  no accounts or sign-ups required. has been the industry's standard dummy text ever 
+                  since the 1500s, 
+                when an unknown printer took a standard dummy text.</p>
+                <div id="skill-text">
+                    <ul>
+                    <li><a class="read-text" href="#">css</a></li>
+                    <li><a class="read-text" href="#">html</a></li>
+                    <li><a class="read-text" href="#">bootstrap</a></li>
+                    <li><a class="read-text" href="#">Ruby</a></li>
+                    </ul>
+                </div>
+                <button type="button" class="div-1--right-btn">See Project</button>
+                </div>
+                </div>
+          <div class="card-2">
+          <div class="card-2-1" id="card-A">
+            <img src="images/card-2-img1.svg" id="img-1" alt="background image">
+            <button type="button" class="div-1--right-btn" id="btn-2">See Project</button> 
+          </div>
+          <div class="card-2-1" id="card-B">
+              <div class="card-B-box">
+              <h4>Data Dashboard 
+                  Healthcare</h4> 
+                  <p class="card-B-para">
+                    A daily selection of privately personalized <br> 
+                    reads;no accounts or sign-ups required.<br>
+                     has been the industry's standard
+                   </p>
+                   <div class="card-B-btns">
+                    <ul>
+                      <li><a class="btns-3" href="#">html</a></li>
+                      <li><a class="btns-3" href="#">bootstrap</a></li>
+                      <li><a class="btns-3" href="#">Ruby</a></li>
+                      </ul>
+                   </div>
```

Comment:
- [ ] [Optional] There is no need to wrap a `<ul>` tag in a `<div>` container, as it is a block-level element.